doc/source/api.rst

```diff
@@ -246,6 +246,24 @@ Testing
 
    test
 
+.. currentmodule:: pandas.plotting
+
+Plotting
+--------
```

Comment:
Can you move this section more to the bottom? There is already a small section on plotting that includes `(de)register_matplotlib_converters`, so you can merge both

Reply:
@jorisvandenbossche Sure thing! Thanks for the feedback. Would it matter if I just did this in a new branch? I'm having trouble crafting the right git command that will un-commit and unstage my changes. Or is it preferred I keep everything on this branch? Or should I just put it in master?

Reply:
Normally you should be able to do `git rm doc/source/savefig/*` and commit those changes. No need to un-commit the previous one (we will squash all commits into one in the end anyway).

Reply:
So if that works, we would prefer that you further push to this branch / PR.

Reply:
Got it. I'll do that now.